doc/book/application-integration/usage-in-a-laminas-mvc-application.md

```diff
@@ -0,0 +1,106 @@
+# Usage in a laminas-mvc Application
+
+The following example shows _one_ potential use case of laminas-eventmanager within
+a laminas-mvc based application.
+The example creates a listener for logging errors of an application with [laminas-log](https://docs.laminas.dev/laminas-log/).
+
+Before starting, make sure laminas-log is [installed and configured](https://docs.laminas.dev/laminas-log/installation/).
+
+laminas-eventmanager is already present in laminas-mvc based applications as it is an event-driven MVC layer based on the event manager.
+
+## Create Listener
+
+Create a listener as class using [a listener aggregate](../aggregates.md) and add the logger via constructor injection to create a valid object with all required dependencies, e.g. `module/Application/src/Listener/ErrorListener.php`:
+
+```php
+namespace Application\Listener;
+
+use Exception;
+use Laminas\EventManager\AbstractListenerAggregate;
+use Laminas\EventManager\EventManagerInterface;
+use Laminas\Log\Logger;
```

Comment:
Should we endorse using a PSR logger? :thinking:

Reply:
The laminas-log component enjoys automatic and seamless integration in a laminas-mvc based application via the laminas-component-installer, so I took these instead of a PSR logger.

Reply:
The question is valid, because I have also been thinking about this topic for a while.